### PR TITLE
CRIU 3.16

### PR DESCRIPTION
**Pull request by adrianreber**
This brings the last few fixups from `criu-dev` over for the 3.16 release.

@checkpoint-restore/maintainers PTAL

If possible I would like to do the 3.16 release this week.

### CODE REVIEW

File: Makefile.versions
Comment (rst0git) [inline Makefile.versions:7]: Is `(TODO)` in the version name intentional?

Comment (adrianreber) [reply Makefile.versions]: No. Removed.

**Comment by rst0git** [comment]
image

**Review by rst0git — APPROVED** [review]

**Review by Snorch — APPROVED** [review]
Looks good.

Note: last time we had some release-tag annotation problem https://github.com/checkpoint-restore/criu/issues/1552 Probably we need to create tag manually or something like this...

**Comment by adrianreber** [comment]
> Looks good.
> 
> Note: last time we had some release-tag annotation problem #1552 Probably we need to create tag manually or something like that...

Thanks for the reminder. I will tag it first locally on my system and then push the tag.

**Comment by avagin** [comment]
Have you prepared a wiki page with release notes?
How is the Jenkins?

**Comment by adrianreber** [comment]
> Have you prepared a wiki page with release notes?

I will do that today. Probably.

> How is the Jenkins?

Pretty good. The ppc64le tests are, however, not running for a over a month now. @mihalicyn any reason ppc64le is not running. Is it still the cgroup race condition problem? Maybe we should a change for criu-dev which runs at least all other tests on ppc64le even if we cannot fix the cgroup problem as it seems to be kernel related.

**Comment by adrianreber** [comment]
@checkpoint-restore/maintainers Can I get a couple of approvals to release 3.16?

**Comment by rst0git** [comment]
@adrianreber would it be possible to add https://github.com/checkpoint-restore/criu/pull/1609 as well?

**Comment by adrianreber** [comment]
> @adrianreber would it be possible to add #1609 as well?

Done

**Comment by avagin** [comment]
@adrianreber could you add a link to the wiki page with this release? As I already said, we need to push all commits in another PR to be sure that all CI-s are happy.

**Comment by adrianreber** [comment]
https://criu.org/Download/criu/3.16

**Pull request merged**
